homeassistant/components/binary_sensor/__init__.py

```diff
@@ -12,14 +12,16 @@
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import STATE_OFF, STATE_ON
 from homeassistant.core import HomeAssistant
-from homeassistant.helpers.config_validation import (  # noqa: F401
+from homeassistant.helpers.config_validation import (
     PLATFORM_SCHEMA,
     PLATFORM_SCHEMA_BASE,
 )
 from homeassistant.helpers.entity import Entity, EntityDescription
 from homeassistant.helpers.entity_component import EntityComponent
 from homeassistant.helpers.typing import ConfigType
 
+__all__ = ["PLATFORM_SCHEMA", "PLATFORM_SCHEMA_BASE"]
```

Comment:
This is incorrect. 

`__all__` must define the entire public API available, this only defines 2 constants. Meaning the rest of this module isn't public anymore.

Reply:
I think that for most entity integrations it would be platform schema, platform schema base, domain, enums and the entity base class

Reply:
Probably :) I think I all for doing that though.